### ISSUE TITLE
Science and Environment

### ISSUE THREAD

**Issue by lc512k** [Financial-Times/next-a11y#16]
DAC-USER-KO-T10-05
Science and Environment https://www.ft.com/science-environment#site-content
Images getting are focus where they shouldn’t or should have a focus ring, as user tabs through stories

Could/should we have a uniform approach to focus landing on 'boxes'? land focus on the entire thing, rather than each bit inside it, tab through a11y.ft.com to see how pa11y dashboard have solved this issue. (relates to https://github.com/Financial-Times/next-a11y/issues/9)

**Comment by andygout** [comment]
Interim solution here: https://github.com/Financial-Times/n-teaser/pull/62

**Comment by lc512k** [comment]
AA issue resolved. Investigate UX around 'focus on the boxes'
